apps/ledger-live-mobile/src/families/polkadot/SimpleOperationFlow/01-Started.tsx

```diff
@@ -4,31 +4,34 @@ import { StyleSheet, View } from "react-native";
 import { useTranslation, Trans } from "react-i18next";
 import { useSelector } from "react-redux";
 import { useTheme } from "@react-navigation/native";
-import type { Transaction } from "@ledgerhq/live-common/generated/types";
+import type {
+  Transaction as PolkadotTransaction,
+  TransactionStatus,
+} from "@ledgerhq/live-common/families/polkadot/types";
 import { getAccountBridge } from "@ledgerhq/live-common/bridge/index";
 import { getMainAccount } from "@ledgerhq/live-common/account/index";
 import useBridgeTransaction from "@ledgerhq/live-common/bridge/useBridgeTransaction";
 import { Button, Alert, Text, Log } from "@ledgerhq/native-ui";
+import { PolkadotAccount } from "@ledgerhq/live-common/families/polkadot/types";
 import { accountScreenSelector } from "../../../reducers/accounts";
 import { ScreenName } from "../../../const";
 import { TrackScreen } from "../../../analytics";
 import TranslatedError from "../../../components/TranslatedError";
 import FlowErrorBottomModal from "../components/FlowErrorBottomModal";
 import SendRowsFee from "../SendRowsFee";
+import type { StackNavigatorProps } from "../../../components/RootNavigator/types/helpers";
+import { PolkadotSimpleOperationFlowParamList } from "./types";
 
-type RouteParams = {
-  transaction: Transaction;
-  mode: "chill" | "withdrawUnbonded" | "setController";
-};
-type Props = {
-  navigation: any;
-  route: {
-    params: RouteParams;
-  };
-};
+type Navigation = StackNavigatorProps<
+  PolkadotSimpleOperationFlowParamList,
+  ScreenName.PolkadotSimpleOperationStarted
+>;
 
 // returns the first error
-function getStatusError(status, type = "errors"): Error | null | undefined {
+function getStatusError(
+  status: TransactionStatus,
+  type: "errors" | "warnings" = "errors",
+): Error | null | undefined {
```

Comment:
I don't think this function can return `undefined`

Reply:
True 👍